lua/diffview/actions.lua

```diff
@@ -338,6 +338,91 @@ local function diff_copy_target(kind)
   end
 end
 
+---@param view DiffView|StandardView
+---@param target "ours"|"theirs"|"base"|"all"|"none"
+local function resolve_all_conflicts(view, target)
+  local main = view.cur_layout:get_main_win()
+  local curfile = main.file
+
+  if main:is_valid() and curfile:is_valid() then
+    local lines = api.nvim_buf_get_lines(curfile.bufnr, 0, -1, false)
+    local conflicts = vcs_utils.parse_conflicts(
+      lines,
+      main.id
+    )
+
+    if next(conflicts) then
+      local content
+      local offset = 0
+      local first
+      local last
+      for _, cur_conflict in pairs(conflicts) do
+
+        -- add offset to line numbers
+        first = cur_conflict.first + offset
+        last = cur_conflict.last + offset
+
+        if target == "ours" then content = cur_conflict.ours.content
+        elseif target == "theirs" then content = cur_conflict.theirs.content
+        elseif target == "base" then content = cur_conflict.base.content
+        elseif target == "all" then
+          content = utils.vec_join(
+            cur_conflict.ours.content,
+            cur_conflict.base.content,
+            cur_conflict.theirs.content
+          )
+        end
+
+        content = content or {}
+        api.nvim_buf_set_lines(curfile.bufnr, first - 1, last, false, content)
+        offset = offset + (#content - (last - first) - 1)
+      end
+
+      utils.set_cursor(main.id, unpack({
+        (content and #content or 0) + first - 1,
+        content and content[1] and #content[#content] or 0
+      }))
```

Comment:
Use the `sync_scroll()` method here, as scroll-bound windows are not synchronized if one of their cursors move while its window is unfocused.

```suggestion
      }))

      view.cur_layout:sync_scroll()
```